app/client/stylesheets/_login.scss

```diff
@@ -1,18 +1,24 @@
 #login {
   height: calc(100% + 49px);
   width: 100%;
-  background-image: url('../assets/images/hero.jpg');
+  background-image: url('../assets/images/login_bg.png'), linear-gradient(to right, #2a2a2a, #191919);
   background-size: cover;
   background-position: center center;
 
   &::before {
     content: '';
-    background-image: linear-gradient(to bottom right, #02ccba 0%, #3d99be 31%, #9848d5 100%);
-    opacity: 0.8;
+    background-image: linear-gradient(to right, #2a2a2a, #191919);
+    animation: fadeout 0.8s ease-in 0.5s forwards;  
+    opacity: 1;
     width: 100%;
     height: calc(100% + 50px);
     position: absolute;
   }
+  
+	@keyframes fadeout {
```

Comment:
we have a global `fadeout` util class: https://github.com/hackuci/quill/blob/develop/app/client/stylesheets/_global.scss#L116